src/tools/rbd/action/Qos.cc

```diff
@@ -0,0 +1,120 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
But, @dillaman should we treat the qos information as a configuration? In my opinion, it's a property of am image, similar with image size. I think we should not treat a image size as a configuration. So I think neither of the properties of capacity(image size) and performance (image iops) should be treat as a configuration. What do you think?